jest.config.ts

```diff
@@ -0,0 +1,14 @@
+import type { Config } from "jest";
+
+const config: Config = {
+    projects: [
+        "<rootDir>/packages/*"
+    ],
+    testRegex: "/tests/*/.*\\.test\\.(ts|tsx)$",
+    testPathIgnorePatterns: ["/node_modules/", "/dist/"],
+    cacheDirectory: "./node_modules/.cache/jest",
+    clearMocks: true,
+    verbose: true
```

Comment:
is verbose: true required, or was this for test purpose?

Reply:
I personally always turn it on. I can remove it if it's an issue for you.

(Event in Orbit we have `verbose: true` and you might have never noticed 😆 )